docs/_static/js/matomo.js

```diff
@@ -1,11 +1,12 @@
-var _paq = window._paq || [];
+var _paq = window._paq = window._paq || [];
```

Comment:
Does this have any effect?

Reply:
not that i can see in this script. maybe just best practice/future proofing.